src/app/profile/certificates/models/certificate.model.ts

```diff
@@ -0,0 +1,13 @@
+import { CertificateTypeEnum } from '../enums/certificate-types.enum';
+
+export interface CertificateModel {
```

Comment:
Is it a `CertificateModel` or `CertificateDTO`?

Reply:
I like Dto more so we're gonna got with that thanks!